ttl/hbp_hippocampus3d_ontology.ttl

```diff
@@ -0,0 +1,396 @@
+### WARNING ###
+### THIS ONTOLOGY IS TEMPORARY DO NOT USE IDENTIFIERS OUTSIDE DEV
+### WARNING ###
+@prefix : <http://www.hbp.FIXME.org/hbp_human_brain_ontology/> .
```

Comment:
change default prefix to http://www.hbp.FIXME.org/hbp_hippocampus3d_brain_ontology